### ISSUE TITLE
Support for disabling auto increment and allowing PK value inserting

### ISSUE THREAD

**Issue by netarc** [imdrasil/jennifer.cr#245]
I have a model that uses a custom primary key (`Int64`) that is set manually. I need a way to disable auto increment which therefore enables the primary key being insertable via `arguments_to_insert`.

I couldn't find a way while digging through docs and code to do this, so I came up with a quick/simple solution below inside the `single_mapping` macro that seems to work great:

```crystal
primary_auto_incrementable = options[:auto] != false && AUTOINCREMENTABLE_STR_TYPES.includes?(stringified_type)
```

Maybe there is something I am not aware of that makes this not work so easily since I am still getting to know the codebase here, but thoughts?

**Comment by imdrasil** [comment]
It seems this should work. ATM I have extremely little time - will come back to this question in 1 week. Meantime you can use `release/0.8.0` (which now includes fix to your previous issue).

**Comment by imdrasil** [comment]
the fix is now in master (0.8.0 version is released)
